internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -143,3 +143,53 @@ func TestCRDHaveValidation(t *testing.T) {
 		})
 	}
 }
+
+func TestStatusDescriptors(t *testing.T) {
+
+	cases := []struct {
+		name       string
+		bundlePath string
+		state      scapiv1alpha2.State
+		function   func(registry.Bundle) scapiv1alpha2.ScorecardTestResult
+	}{
+		{
+			name:       "Ill formatted CR",
+			bundlePath: "../testdata/statusdescriptor/error_bundle",
+			state:      scapiv1alpha2.ErrorState,
+			function:   StatusDescriptorsTest,
+		},
+		{
+			name:       "Missing status in CR",
+			bundlePath: "../testdata/statusdescriptor/invalid_status_bundle",
+			state:      scapiv1alpha2.FailState,
+			function:   StatusDescriptorsTest,
+		},
+		{
+			name:       "Missing owned CRD in CSV",
```

Comment:
```suggestion
                         // It will check if the field spec.customresourcedefinitions.owned.statusDescriptors exists 
			name:       "Should fail when CSV has not statusDescriptors for the Owned CRD",
```